packages/@aws-cdk/aws-efs/package.json

```diff
@@ -89,6 +89,7 @@
     "@aws-cdk/aws-ec2": "0.0.0",
     "@aws-cdk/aws-kms": "0.0.0",
     "@aws-cdk/cx-api": "0.0.0",
```

Comment:
Does this still need to depend on cx-api?

Same for any other construct library package that depends on it.

Reply:
Yeah because some of the constants it uses are still in `cx-api`. I'll double check though.